fuel-core/src/cli/run/p2p.rs

```diff
@@ -40,10 +40,19 @@ pub struct P2pArgs {
     pub max_block_size: usize,
 
     /// Addresses of the bootstrap nodes
-    /// They should contain PeerId at the end of the specified Multiaddr
+    /// They should contain PeerId within their `Multiaddr`
     #[clap(long = "bootstrap_nodes")]
     pub bootstrap_nodes: Vec<Multiaddr>,
 
+    /// Addresses of the reserved nodes
+    /// They should contain PeerId within their `Multiaddr`
+    #[clap(long = "reserved_nodes")]
+    pub reserved_nodes: Vec<Multiaddr>,
+
+    /// Set to true if you only want to connect to the reserved nodes
+    #[clap(long = "reserved_nodes_only_mode")]
+    pub reserved_nodes_only_mode: bool,
```

Comment:
I updated the description a bit, but don't want to be too much specific because there are use cases for it outside of PoA nodes, and the reasoning is mainly straightforward -> you don't want it to be connected to any other node (for whatever reason)